### PR TITLE
feat(python): updates python support

### PR DESCRIPTION
**Pull request by invincibleJai**
- updates python support

### CODE REVIEW

File: src/Utils.ts [hunk 1]
```diff
@@ -21,6 +21,6 @@ export namespace Utils {
     const pypiPath: string = vscode.workspace
       .getConfiguration('python')
       .get('pythonPath');
-    return pypiPath;
+    return pypiPath ? `"${pypiPath}"` : 'npm';
```
Comment (sivaavkd) [inline src/Utils.ts]: did you mean 'python' instead of 'npm' ?

File: test/utils.test.ts [hunk 2]
```diff
@@ -24,7 +24,12 @@ suite('Utils module', () => {
...
   test('getNodeExecutable should return npm', () => {
-    let mavenPath = Utils.getNodeExecutable();
-    expect(mavenPath).equals('npm');
+    let npmPath = Utils.getNodeExecutable();
+    expect(npmPath).equals('npm');
+  });
+
+  test('getPypiExecutable should return python', () => {
+    let pypiPath = Utils.getPypiExecutable();
+    expect(pypiPath).equals('"python"');
```
Comment (sivaavkd) [inline test/utils.test.ts]: python is enclosed in double brackets and npm is not.  Is this deliberate ?

Comment (invincibleJai) [reply test/utils.test.ts]: for now yes, as test are failing without that. See if you can get it work

**Comment by invincibleJai** [comment]
[test]

**Comment by invincibleJai** [comment]
[test]

File: test/utils.test.ts (hunk 2 above)
Comment (invincibleJai) [reply test/utils.test.ts]: I have fixed it

**Comment by sivaavkd** [comment]
LGTM

**Comment by arajkumar** [comment]
https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/blob/c9ce88018bae09a45ca0b3c187d61593fbc4831e/src/statusMessages.ts#L19

Probably we should consider to clean up & keep the above snippet as a separate python file.

**Pull request merged**